inst/rmarkdown/templates/pdf/skeleton/pinp.cls

```diff
@@ -178,11 +178,15 @@
 \setlength{\affilsep}{8.5pt} % 16.5pts between base of author line and base of affil line
 \renewcommand\Authfont{\color{color0}\normalfont\sffamily\bfseries\fontsize{9}{11}\selectfont}
 \renewcommand\Affilfont{\color{color0}\normalfont\sffamily\fontsize{7}{8}\selectfont}
-\makeatletter
+
 \renewcommand\AB@affilsepx{; \protect\Affilfont}
-\makeatother
+
 \renewcommand\Authands{, and }
 
+%% Choose template type
+\newcommand*{\templatetype}[1]{%
```

Comment:
I think we also dropped this.

Reply:
Ok

Reply:
Good catch. I even tried being cute and leaving it in for "advanced" users but on `pnasinvited` (after copying the `.sty` in) in balks on a redefinition, and on `pnasmatematics` ... nothing happens (for the one-col test file I used).